packages/components/src/utils/accessibilityProfile/typeguards.ts

```diff
@@ -0,0 +1,7 @@
+import { AccessibilityProfile } from '../../types';
+
+export function isAccessibilityProfile(
+  value: string
+): value is AccessibilityProfile {
+  return Object.values<string>(AccessibilityProfile).includes(value);
+}
```

Comment:
```suggestion
export function isAccessibilityProfile(
  value: unknown
): value is AccessibilityProfile {
  return Object.values(AccessibilityProfile).includes(value as AccessibilityProfile);
}
```

To make it easier to use the type guard with any input type instead of just strings.